modules/pam_pwhistory/pam_pwhistory.c

```diff
@@ -60,6 +65,7 @@
 #include "pam_inline.h"
 
 #define DEFAULT_BUFLEN 2048
+#define MAX_FD_NO 20000
```

Comment:
MAX_FD_NO is unused.

Reply:
DEFAULT_BUFLEN was also unused so I've deleted both of them.